### PR TITLE
Update use of Object.assign on Arrays

### PR DESCRIPTION
**Pull request by clint77**
Update use of Object.assign on Arrays

### CODE REVIEW

**Comment by clint77** [comment]
I signed it!

File: docs/_includes/projects/properties/defaultconverter/my-element.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class MyElement extends LitElement {
     this.prop1 = randy.toString();
     this.prop2 = randy;
     this.prop3 = !myBool;
-    this.prop4 = Object.assign([], [...this.prop4], randy);
+    this.prop4 = Object.assign([...this.prop4, randy]);
```
Comment (keanulee) [inline docs/_includes/projects/properties/defaultconverter/my-element.js]: `[...this.prop4, randy]` will already create a new array, so `Object.assign()` is unnecessary.

```js
this.prop4 = [...this.prop4, randy];
```

Comment (clint77) [reply docs/_includes/projects/properties/defaultconverter/my-element.js]: Yes, `this.prop4 = [...this.prop4, randy];` is what we need here. How do I change this in the pull request? Do I need to close this and open another one with the changes?

Comment (qqiao) [reply docs/_includes/projects/properties/defaultconverter/my-element.js]: Pushing to the same branch ought to do it?

Comment (clint77) [inline docs/_includes/projects/properties/defaultconverter/my-element.js]: As per comments above by @keanulee 

```suggestion
    this.prop4 = [...this.prop4, randy];
```

Comment (clint77) [reply docs/_includes/projects/properties/defaultconverter/my-element.js]: I push a commit as per the suggested change by @keanulee. Thanks!

**Comment by keanulee** [comment]
LGTM, just had to update in one more place.

CLAs are ok - I'm the only other commit author.

**Pull request merged**